tests/test_resourcegroupstaggingapi/test_resourcegroupstaggingapi.py

```diff
@@ -759,6 +759,32 @@ def test_get_resources_sns():
         "Tags"
     ]
 
+@mock_aws
+def test_get_resources_ssm():
+    import json
+    import pkgutil
+    import yaml
+
+    template_file = pkgutil.get_data(__name__, "test_templates/good.yaml")
```

Comment:
Can we reuse this from the SSM tests?

I think we can just do a toplevel import:
`from tests.test_ssm/test_ssm_docs import _get_yaml_template`

That way we don't have to duplicate anything

Reply:
hi, i've changed it to the toplevel import in my latest commit